### PR TITLE
Events main page

### PR DESCRIPTION
**Pull request by alanmoo**
Styles of main page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,8 @@
     "json-loader": "^0.5.4",
     "live-server": "^0.9.2",
     "mofo-style": "^2.4.0",
+    "moment": "^2.13.0",
```
Comment (gvn) [inline package.json]: Lock these 2 down

**Comment by gvn** [comment]
When I open http://127.0.0.1:1818/programs/events I get this in the console:

```
compiled.js:63466 TypeError: Cannot read property 'timeZone' of null(…)
```

File: app/react/components/event-list/event-list.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import DataCard from "../../components/data-card/data-card.jsx";
+import Moment from "moment-timezone";
+import {Link} from "react-router";
+
+export default React.createClass({
+  timeZone: ``,
```
Comment (gvn) [inline app/react/components/event-list/event-list.jsx]: Shouldn't this be on `state`?

Comment (alanmoo) [inline app/react/components/event-list/event-list.jsx]: It's not something that changes, so I just wanted to set it once in componentDidMount, though I can see moving it to state. I was also evaluating moving it up the project and using it as a context somehow, or just moving the timezone/event date calculation logic into the `Service` module

**Comment by gvn** [comment]
R+!

**Pull request merged**